src/vm/threadsuspend.cpp

```diff
@@ -4987,10 +4984,19 @@ HRESULT ThreadSuspend::SuspendRuntime(ThreadSuspend::SUSPEND_REASON reason)
             {
                 FastInterlockOr((ULONG *) &thread->m_State, Thread::TS_GCSuspendPending);
                 countThreads++;
+
+#if defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
```

Comment:
Is it correct to have this here? This code is for disabled thread suspension and using polling instead. Why do we inject the suspension here then?